lib/charms/ovn_charm.py

```diff
@@ -1127,7 +1127,8 @@ def configure_ovs(self, sb_conn, mlockall_changed):
                            'external-ids:system-id={}'
                            .format(self.get_ovs_hostname()),
                            'external-ids:ovn-remote={}'.format(sb_conn),
-                           'external_ids:ovn-match-northd-version=true',
+                           'external_ids:ovn-match-northd-version={}'
```

Comment:
I didn't find explicit list of accepted values for this option in OVN docs, but [Upgrade guide](https://docs.ovn.org/en/stable/intro/install/ovn-upgrades.html#fail-safe-upgrade) and [man page (`ctrl+f ovn-match-northd-version`)](https://www.ovn.org/support/dist-docs/ovn-controller.8.html) mention only lowercase `true`/`false`.

Turning boolean into string will produce capitalized `True`/`False`, do you know if these value will work as well?

Reply:
Excellent question, I did check and they do a case insensitive match on 'true' or 'false' in the code, ref:
https://github.com/ovn-org/ovn/blob/5d4b8038b6aca98bda0a4b22eb1471c87ac24b51/controller/ovn-controller.c#L4616
https://github.com/ovn-org/ovn/blob/5d4b8038b6aca98bda0a4b22eb1471c87ac24b51/lib/ovn-util.c#L1081
https://github.com/openvswitch/ovs/blob/e3d5616706107b4c2bdc483359cf1854611cd9e0/lib/smap.c#L224

Reply:
Thanks for looking that up :+1: